package.json

```diff
@@ -102,6 +102,7 @@
     "@angularclass/conventions-loader": "1.0.2",
     "@angularclass/hmr": "1.2.0",
     "@angularclass/hmr-loader": "3.0.2",
+    "@types/lodash": "^4.14.50",
```

Comment:
Types are for converting TypeScript to JavaScript and are only needed when we compile it, not run it.